src/Generator/OutputGeneratorTrait.php

```diff
@@ -60,7 +60,7 @@ protected function createResponseDenormalizationStatement($status, $schema, Cont
                         new Expr\PropertyFetch(new Expr\Variable('this'), 'serializer'),
                         'deserialize',
                         [
-                            new Arg(new Expr\MethodCall(new Expr\MethodCall(new Expr\Variable('response'), 'getBody'), 'getContents')),
+                            new Arg(new Expr\Cast\String_(new Expr\MethodCall(new Expr\Variable('response'), 'getBody'))),
```

Comment:
I think this is better. I hate calling toString. It doesn't conform PSR-2. Can't it be null as well?